src/stroke.rs

```diff
@@ -502,7 +505,7 @@ impl StrokeCtx {
         let c2 = p[3] - 3.0 * p[2] + 3.0 * p[1] - p[0];
         let roots = solve_quadratic(c0, c1, c2);
         // discard cusps right at endpoints
-        const EPSILON: f64 = 1e-12;
+        const EPSILON: f64 = 1e-6;
```

Comment:
I wonder if it's worth spending some time rationalizing all the accuracy parameters sometime (saying why they are the value they are, or if they are arbitrary).